src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  --main--color--: #fff;
+  --for--links--: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: var(--main--color--);
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  color: #000;
+  height: 60px;
+  text-decoration: none;
+  white-space: nowrap;
+  font-family: "Roboto", sans-serif;
```

Comment:
3'rd point in checklist

Reply:
i dont understand what i need to do there.I have font family in nav__links . I need to put "font - family " into another selector?for exm into : hmtl ?

Reply:
just add `Arial` or `Helvetica` as a fallback fonts like this - `"Roboto", Helvetica, sans-serif;` because it may be a situation when for some reason Roboto didn't work